Elemental/Components/AeTable/AePaginatedTable.razor

```diff
@@ -108,4 +107,11 @@
         LoadPageItems(true);
     }
 
+    private void CheckOutOfBounds()
+    {
+        int PageCount = (int)Math.Ceiling(Total / (decimal)Math.Max(1, PageSize));
+        if (Page > PageCount - 1) Page = PageCount - 1;
+        else if (Page < 0) Page = 0;
+    }
+
```

Comment:
For consistency we should use the implemented interface's `PageCount` ([here](https://github.com/apption-oscode/elemental-blazor/blob/15cb40c4eeec58419aec63a28fc1434541ff61c3/Elemental/Components/AePagination/IPageableComponent.cs#L10)) that's already defined

I know it's tempting to use the inline if/else but let's keep the curly braces

Reply:
Ok, in the future I'll note to use curly braces. 👍